api/proxy/servers/arbitrum_altda/cli.go

```diff
@@ -0,0 +1,51 @@
+package arbitrum_altda
+
+import (
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	ListenAddrFlagName = "arbitrum-da.addr"
+	PortFlagName       = "arbitrum-da.port"
+	Enabled            = "arbitrum-da.enabled"
+)
+
+func withEnvPrefix(prefix, s string) []string {
+	return []string{prefix + "_ARB_CUSTOM_DA_" + s}
+}
+
+func CLIFlags(envPrefix string, category string) []cli.Flag {
+	flags := []cli.Flag{
+		&cli.StringFlag{
+			Name:     ListenAddrFlagName,
+			Usage:    "Server listening address",
+			Value:    "0.0.0.0",
+			EnvVars:  withEnvPrefix(envPrefix, "ADDR"),
+			Category: category,
+		},
+		&cli.IntFlag{
+			Name:     PortFlagName,
+			Usage:    "Server listening port",
+			Value:    3100,
```

Comment:
same port as the rest server right? don't think 2 services can listen on a single port

Reply:
[e269999](https://github.com/Layr-Labs/eigenda/pull/1957/commits/e269999c94bc2261e4d890cfb4b705f45b830c17)